test/channels/test_manifest_validator.py

```diff
@@ -0,0 +1,102 @@
+from tempfile import SpooledTemporaryFile
+from unittest.mock import patch
+
+import pytest
+
+from operationsgateway_api.src.channels.channel_manifest import ChannelManifest
+from operationsgateway_api.src.exceptions import ChannelManifestError
+
+
+class TestManifestValidator:
+    @pytest.mark.asyncio
+    async def test_channel_removed(self):
+
+        spooled_file = SpooledTemporaryFile()
+        content = (
+            '{"_id": "19830222132431", "channels": {"PM-201-FE-CAM-1": '
+            '{"name": "D100 front-end NF", "path": "/PM-201/FE", "type": "image"}, '
+            '"PM-201-FE-CAM-2": {"name": "D100 front-end FF", "path": "/PM-201/FE",'
+            ' "type": "image"}}}'
+        )
```

Comment:
Similar to my previous comment, could this use dictionaries please? There's a couple of places in this file where this happens, so please make the changes to all the instances in this file :)